src/lib/edge-functions/registry.ts

```diff
@@ -126,6 +126,10 @@ export class EdgeFunctionsRegistry {
         throw new Error('Build error')
       }
 
+      if (this.functions.length === 0) {
```

Comment:
If we don't want to print the warning when there are no functions (which I agree we shouldn't do), should we have a check inside `processGraph`, next to the code that prints the warning? That way we'll handle a correct behaviour regardless of where we call `processGraph` from, otherwise it's up to the caller to ensure that.